BungieFrameWork/BFW_Source/BFW_LocalInput/Platform_Win32/BFW_LI_Platform_Win32.h

```diff
@@ -1,3 +1,4 @@
+#if UUmPlatform == UUmPlatform_Win32
```

Comment:
Ditto; it shouldn't be necessary to include these in the headers anyway as the platform-specific headers are only going to be included into the platform specific code (i.e. BFW_LI_Platform_Win32.h is included in BFW_LI_Platform_Win32.c, which we wouldn't be compiling with Linux/macOS as the target).